src/snowcli/cli/nativeapp/commands.py

```diff
@@ -62,7 +62,7 @@ def app_init(
     **options,
 ) -> CommandResult:
     """
-    Initialize a Native Apps project, optionally with a --template-repo and a --template.
+    Initializes a Native Apps project.
```

Comment:
This makes sense @sfc-gh-mheavin . His explanation being - we dont list options in the description of other commands, and we should stick to that here as well. I agree with this.